events2.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let input = document.getElementById('user-input');
+  let ul = document.getElementById('ul');
+  let submit = document.getElementById('submit');
+  let p = document.getElementById('p');
+  submit.addEventListener('click', e => {
+    e.preventDefault();
+    if (input.value === '') {
+      p.innerText = 'Error!';
+    } else {
+      let li = document.createElement('li');
+      li.addEventListener('click', e => {
+        li.style.textDecoration = 'line-through';
+      });
+      li.innerText = input.value;
+      ul.appendChild(li);
+      input.value = '';
+    }
+  });
+});
```

Comment:
Nice job with this assignment Muna! 

Side note for something that's not in the instructions at all but I thought would be useful for the future:

- Currently your code is adding strings composed of empty spaces to the list. You can avoid this by using the [String.prototype.trim() method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) to check for and remove the beginning/ending empty spaces in a string. So your line 8 would look like so:
```
if (input.value === '' || !input.value.trim()) {
```